src/vs/platform/actionWidget/browser/actionWidget.css

```diff
@@ -3,7 +3,7 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-.codeActionWidget {
+.actionWidget {
```

Comment:
nit: I know it was like this before, but most classes are done like `.action-widget` (eg. `.monaco-list`, `.monaco-workbench`, etc.)